.github/workflows/scripts/kubernetes/core/run-quickstart-test-on-kubernetes.sh

```diff
@@ -76,49 +76,23 @@ fi
 
 ################################################################################################
 # Install any pre-requisites. Function is from overridable-functions.sh
+
 echo "Checking if we need to install pre-requisites"
 installPrerequisites "${application}"
 
 ################################################################################################
 # Provision server and push imagestream
 
-echo "Building application and provisioning server image..."
-mvn -B package -Popenshift wildfly:image -DskipTests
-
-echo "Tagging image and pushing to registry..."
-export root_image_name="localhost:5000/${application}"
-export image="${root_image_name}:latest"
-docker tag ${qs_dir} ${image}
-docker push ${image}
+provisionServer "${application}" "${qs_dir}"
 
 ################################################################################################
-# Helm install, waiting for the pods to come up
```

Comment:
Same observation as for the running of the tests. IMO the default behaviour should remain in this file, and a check should be done for when we don't want to do that, and handle that accordingly.

Reply:
Thanks for your review, @kabir. I hope the last commit is close enough to having a default behaviour. Please, let me know what you think about it :-)